services/src/main/scala/cromwell/services/auth/ecm/EcmConfig.scala

```diff
@@ -3,8 +3,8 @@ package cromwell.services.auth.ecm
 import com.typesafe.config.Config
 import net.ceedubs.ficus.Ficus._
 
-final case class EcmConfig(baseUrl: String)
+final case class EcmConfig(baseUrl: Option[String])
```

Comment:
TOL is there a reason the `enabled` and `auth.azure` fields are not here?

Reply:
Discussed via Slack but posting here for posterity: Since these values are more related to enabling `GithubAuthVending` service they are separate from the `ecm` sub-section in the config.